classes/mapworker.php

```diff
@@ -110,6 +118,21 @@ public function replace_stylesheet(array $placestoreoverride = []) : void {
         $this->load_dom();
     }
 
+    /**
+     * Replaces the svg defs
```

Comment:
For someone who is not really familiar with svg (like me), it would be helpful to have a little description of what the defs are, and what they are for